atomic_reactor/plugins/pre_add_labels_in_df.py

```diff
@@ -126,6 +131,28 @@ def generate_auto_labels(self, auto_labels):
         docker_info = self.tasker.get_info()
         generated['com.redhat.build-host'] = docker_info['Name']
 
+        info_url = self.info_url_format
+        labels = Labels(self.labels)
+        find_subs = re.finditer('{([^}]+)}', info_url)
```

Comment:
`kwargs` would not be a list of labels, but a dict mapping label name to label value. This is retrieved by `DockerfileParser.labels`. The only reason I can think of for not using that is if we want to support automatic translation of old-style to new-style labels. This to me seems unnecessary as the `info_url_format` is configurable.

```python
>>> from dockerfile_parse import DockerfileParser
>>> parser = DockerfileParser()
>>> parser.labels
{u'foo': u'oof', u'spam': u'maps'}
>>> 'registry.com/{foo}/{spam}'.format(**parser.labels)
'registry.com/oof/maps'
>>> 
```